tests/test_modeling_common.py

```diff
@@ -398,7 +398,9 @@ class CopyClass(base_class):
                 model_slow_init = base_class_copy.from_pretrained(tmpdirname, _fast_init=False)
 
                 for key in model_fast_init.state_dict().keys():
-                    max_diff = (model_slow_init.state_dict()[key] - model_fast_init.state_dict()[key]).sum().item()
```

Comment:
We can sum the values, as the positive/negative values will cancel out. This is not reliable.